R/with_parameters.R

```diff
@@ -27,9 +27,10 @@
 #' data frame, pass it in the `.cases` argument.
 #'
 #' One parameter is noteworthy. If the user passes a character vector as
-#' `test_name`, each instance is combined with `desc_stub` to create the
+#' `.test_name`, each instance is combined with `desc_stub` to create the
 #' completed test name. Similarly, the named argument from `cases()` is combined
-#' with `desc_stub` to create the parameterized test names.
+#' with `desc_stub` to create the parameterized test names. When names aren't
+#' provided, they will be automatically generated using the test data.
```

Comment:
would help to mention how it's constructed so user can more quickly decide if they're OK w auto-naming

Reply:
Great point.